raiden/transfer/mediated_transfer/mediation_fee.py

```diff
@@ -117,6 +121,64 @@ def reversed(self: T) -> T:
         self._update_penalty_func()
         return reversed_instance
 
+    @staticmethod
+    def mediation_fee_func(
+        schedule_in: "FeeScheduleState",
+        schedule_out: "FeeScheduleState",
+        balance_in: Balance,
+        balance_out: Balance,
+        capacity_in: TokenAmount,  # TODO: rename
+        amount_with_fees: PaymentWithFeeAmount,
+        cap_fees: bool,
+    ) -> Interpolate:
+        if amount_with_fees > capacity_in:
```

Comment:
Isn't this checked before?

Reply:
No in some pathological tests IIRC. We might be able to remove it when improving the tests.